db/schema.rb

```diff
@@ -310,6 +310,7 @@
     t.datetime "processed_at"
     t.boolean "automatic_notifications", default: false
     t.boolean "mails_sent", default: false
+    t.datetime "opened_at"
```

Comment:
I have one quick question, just to confirm that understand the logic :)
if `opened_at` is `null`, this means that the webhook endpoint is not contacted yet.
Once all the webhook endpoints are contacted, we update this field.
is that correct?

Reply:
Almost. Once we recognised that the order cycle opened and we dispatched events then `opened_at` is set. So the background jobs to deliver webhooks are enqueued at this point but may not be executed yet. It happens in parallel and the endpoints are contacted at the same time or after the `opened_at` field is set.

Reply:
I understand better now.
Thank you